lib/cleaned_output.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'colorize'
+
+class CleanedOutput
+  def initialize(output:, morph_api_key: ENV['MORPH_API_KEY'])
+    @output = output
+    @morph_api_key = morph_api_key
+  end
+
+  def to_s
+    (cleaned_output[-64_000..-1] || cleaned_output).uncolorize
```

Comment:
I suspect this would be a little clearer if the truncation code were pulled out into its own method.